### PR TITLE
Added route service in order to manage it in a separate service

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Moved static routes management from dhcpd service to routes service

# Impacts
Static routes

# Issue
In case of a webauth layer 3 network when the dhcp service has been deactivated there is no way to manage static routes.

# Delete branch after merge
YES

# NEWS file entries
## New Features
* New routes service to manage statics routes

### CODE REVIEW

**Comment by julsemaan** [comment]
Looked at the code, and it seems all right

Will test it a bit and merge

**Comment by fdurand** [comment]
Change has been done, ready for a review

**Comment by fdurand** [comment]
Bump

File: lib/pf/services/manager/routes.pm [hunk 1]
```diff
@@ -88,8 +89,8 @@ sub isAlive {
             $route_exist = $network;
         }
     }
-
-    my $routes_applied = defined(pf_run("route | grep ".$route_exist));
+    my $routes_applied = 0;
```
Comment (julsemaan) [inline lib/pf/services/manager/routes.pm]: $FALSE

**Comment by julsemaan** [comment]
Will attempt to test today

**Comment by julsemaan** [comment]
Did a few tests and it seems to be behaving correctly.

**Pull request merged**